app/screens/settings/notification_settings_email/notification_settings_email_base.js

```diff
@@ -23,6 +23,34 @@ export default class NotificationSettingsEmailBase extends PureComponent {
         theme: PropTypes.object.isRequired,
     };
 
+    static getDerivedStateFromProps(props, state) {
```

Comment:
Is there a way to avoid `gerDerivedStateFromProps`?

Reply:
Ok, avoided.